packages/components/Molecules/AnchorLinks/index.vue

```diff
@@ -6,6 +6,7 @@
     <div class="rpl-anchor-links__row">
       <ul class="rpl-anchor-links__items" v-if="links">
         <li class="rpl-anchor-links__item" v-for="(item, index) of links" :key="index">
+          <sup v-if="item.type && item.type === 'h3'" class="rpl-anchor-links__item-sup">L</sup>
```

Comment:
Hey @rubyllamadora Do you think this can be a nested list and using `list-style-image`? When I test it in screen reader, it read the `L` as well.

Reply:
😬 I'll switch this to image. Thanks for testing @tim-yao I'll remember to consider screen reader next time.